multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/variables/Variables.java

```diff
@@ -589,4 +593,16 @@ public Duration deserialize(Object serializedObject) {
                                                                                        .defaultValue(true)
                                                                                        .build();
 
+    ImmutableJsonStringListVariable<List<Resource>> BATCHES_TO_PROCESS = ImmutableJsonStringListVariable.<List<Resource>> builder()
```

Comment:
Use just the interface here -> `Variable<List<Resource>>`
And shouldn't this be `Variable<List<List<Resource>>>`?